app/services/group_service.rb

```diff
@@ -42,6 +42,6 @@ def self.update(group:, params:, actor:)
   def self.archive(group:, actor:)
     actor.ability.authorize! :archive, group
     group.archive!
-    EventBus.broadcast('group_archive', group, actor)
+    EventBus.broadcast('group_archive', group)
```

Comment:
We should keep this line here, because there may be some other thing that comes along which needs to know about who archived the group.